src/content/en/tools/chrome-devtools/profile/network-performance/understanding-resource-timing.markdown

```diff
@@ -0,0 +1,222 @@
+  <dd>
+    Time spent performing the DNS lookup.
+    Every new domain on a page requires a full roundtrip to do the DNS lookup.
+  </dd>
+
+  <dt class="initial-connection"><strong> Initial Connection / Connecting</strong></dt>
+  <dd>Time it took to establish a connection, including <abbr title="Transmission Control Protocol">TCP</abbr> handshakes/retries and negotiating a <abbr title="Secure Sockets Layer">SSL</abbr>.</dd>
+
+  <dt class="ssl"><strong> SSL</strong></dt>
+  <dd>Time spent completing a SSL handshake.</dd>
+
+  <dt class="request-sent"><strong> Request Sent / Sending</strong></dt>
+  <dd>
+    Time spent issuing the network request.
+    Typically a fraction of a millisecond.
+  </dd>
+
+  <dt class="ttfb"><strong> Waiting (<abbr title="Time To First Byte">TTFB</abbr>)</strong></dt>
+  <dd>
+    Time spent waiting for the initial response, also known as the Time To First Byte.
+    This time captures the latency of a round trip to the server in addition to the time spent waiting for the server to deliver the response.
+  </dd>
+
+  <dt class="content-download"><strong> Content Download / Downloading</strong></dt>
+  <dd>Time spent receiving the response data.</dd>
+</dl>
+
+
+## Diagnosing Network Issues
+
+There are numerous possible issues that can be uncovered through the network panel.
+Being able to find these requires a good understanding of how clients and servers communicate and the limitations imposed by the protocols.
+
+### Queued or Stalled series
+
+The most common issue seen is a series of items that are queued or stalled.
+This indicates that too many resources are being retrieved from a single client.
+On HTTP 1.0/1.1 connections, Chrome enforces a maximum of six TCP connections per host.
+If you are requesting twelve items at once, the first six will begin and the last half will be queued.
+Once one of the original half is finished, the first item in the queue will begin its request process.
+
+![Stalled series of requests](imgs/stalled-request-series.png)
+
+To fix this problem for traditional HTTP 1 traffic, you would need to implement domain sharding.
+That is making multiple subdomains on your application to serve resources from.
+Then split the resources being served evenly among the subdomains.
+
+The fix for HTTP 1 connections does **not** apply to HTTP 2 connections.
+In fact, it hurts them. If you have HTTP 2 deployed, don’t domain-shard your resources as it works against how HTTP 2 is engineered to operate.
+In HTTP 2, there is a single TCP connection to the server that acts as a multiplexed connection.
+This gets rid of the six connection limit of HTTP 1 and multiple resources can be transferred over the single connection simultaneously.
+
+### Slow Time to First Byte
+
+<small>AKA: lots of green</small>
+
+![High TTFB Indicator](imgs/indicator-of-high-ttfb.png)
+
+A slow time to first byte is recognized by a high waiting time.
```

Comment:
Put TTFB in parentheses after `A slow time to first byte` so that people know the acronym.

`A slow time to first byte (TTFB) ...`